src/client/localplayer.cpp

```diff
@@ -497,6 +497,7 @@ void LocalPlayer::applyControl(float dtime, Environment *env)
 	bool pitch_move = (free_move || in_liquid) && player_settings.pitch_move;
 	// When aux1_descends is enabled the fast key is used to go down, so fast isn't possible
 	bool fast_climb = fast_move && control.aux1 && !player_settings.aux1_descends;
+	bool continuous_forward = player_settings.continuous_forward && !isDead();
```

Comment:
```suggestion
```

Reply:
Where did it go? o_O

This was used in two places IIRC.

Reply:
https://github.com/minetest/minetest/commit/9d09c87f13a197672bd69ede8de987953af945ad

Oh right :D

Reply:
Fixed, thanks